x-pack/plugins/upgrade_assistant/public/application/components/overview/upgrade_step/upgrade_step.tsx

```diff
@@ -125,13 +122,13 @@ const UpgradeStep = () => {
   } else {
     callToAction = (
       <EuiButton
-        href={docLinks.links.upgrade.upgradingElasticStack}
+        href={docLinks.links.upgrade.upgradingStackOnPrem}
         target="_blank"
         data-test-subj="upgradeSetupDocsLink"
         iconSide="right"
         iconType="popout"
       >
-        {i18nTexts.upgradeStepLink}
+        {i18nTexts.upgradeGuideLink}
```

Comment:
It seemed strange that we used different copy for the doc link on-prem ("Learn more") vs. cloud. Now the copy is the same in both places - "View upgrade guide".